modules/ebs__explore_snapshots/main.py

```diff
@@ -7,6 +7,8 @@
 
 from botocore.exceptions import ClientError
 
+from pacu.io import print
```

Comment:
Print can just be imported from pacu.io now, no need to pass it around with the pacu_main object.